src/Sylius/Bundle/ApiBundle/Command/AddProductReview.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/** @experimental */
+class AddProductReview
+{
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $title;
+
+    /**
+     * @var int
+     * @psalm-immutable
+     */
+    public $rating;
+
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $comment;
+
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $product;
```

Comment:
```suggestion
    public $productCode;
```
as we should use iri only to get a unique identifier for given resource